Diablo Edit2/DlgCharItems.cpp

```diff
@@ -162,7 +167,7 @@ const int POSITION_INFO[POSITION_END][5] = {
 static CRect PositionToRect(EPosition pos) {
 	ASSERT(pos < POSITION_END);
 	auto & p = POSITION_INFO[pos];
-	return CRect(p[0], p[1], p[0] + GRID_WIDTH * p[2], p[1] + GRID_WIDTH * p[3]);
+	return CRect(p[0] * DLG_SCALE_X, p[1] * DLG_SCALE_Y, p[0] * DLG_SCALE_X + GRID_WIDTH_SCALED * p[2], p[1] * DLG_SCALE_Y + GRID_WIDTH_SCALED * p[3]);
```

Comment:
`C4244: conversion from 'double' to 'int', possible loss of data`

Reply:
I missed this warning with other similar warnings in compiling, but the warning is not really valid for here as we are taking the floor of that operation always.  So we can add (LONG) to the multiplication.